pages/docs/tutorials/native/mpp-ios-android.md

```diff
@@ -0,0 +1,566 @@
+    into frameworkDir
+    
+    doLast {
+        new File(frameworkDir, 'gradlew').with {
+            text = "#!/bin/bash\nexport 'JAVA_HOME=${System.getProperty("java.home")}'\ncd '${rootProject.rootDir}'\n./gradlew \$@\n"
+            setExecutable(true)
+        }
+    }
+}
+
+tasks.build.dependsOn packForXCode
+```
+</div>
+
+Let's switch back to the Android Studio and execute the `build` target of the `SharedCode` project from
+the *Gradle* tool window. The task checks for environment variables set by the Xcode build and copies
+the right variant of the framework into the `SharedCode/build/xcode-frameworks` folder. We include the
+framework from the folder into the build:
+
+### Setting up Xcode Dependency for iOS Emulator
+
+Let's add the `SharedCode` framework to the Xcode project.
+For that let's click on the root node of the *project navigator* and select the *target* settings.
+Next, we click on the `+` in the *Embedded Binaries* section, click *Add Other...* button in the dialog
+to choose the framework from the disk. We point to the following folder: 
+```
+SharedCode/build/xcode-frameworks/SharedCode.framework
+```
+
+You shall see something similar to this: 
+![Xcode General Screen]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-general.png') }})
+
+We need to disable the *Bitcode* feature in the project too. Kotlin/Native produces the fully native
+binaries, not the LLVM bitcode, so we switch to the *Build Settings* tab and type `bitcode` into
+the search field. Select `No` for the *Enable Bitcode* option.
+
+![Xcode Build Settings]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-bitcode.png') }})
+
+Now we need to explain Xcode, where to look for frameworks. We need to add the full path to the 
+`SharedCode/build/xcode-frameworks` into the *Search Paths | Framework Search Paths* section.
+
+![Xcode Build Settings]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-search-path.png') }})
+
+The final step is to make Xcode call our Gradle build to prepare the `SharedCode` framework.
+We open the *Build Phases* tab and click `+` to add the *New Run Script Phase* 
+
+Let's add the following code into it:
+
+<div class="sample" markdown="1" mode="bash" theme="idea" data-highlight-only="1" auto-indent="false">
+
+```bash
+cd $SRCROOT/../../SharedCode/build/xcode-frameworks
+./gradlew :SharedCode:packForXCode
+```
+</div>
+
+Note, the `$SRCROOT/../..` is the way to find the project folder root from the Xcode project sources root.
+It may depend on the way how you created your project in Xcode. 
+
+![Xcode Build Phases]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-run-script.png') }})
+
+You should drag the created build phase to the top of the list.
+
+![Xcode Build Phases]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-run-script-order.png') }})
+
+We are ready now to start coding the iOS application and to use Kotlin code from it.
+
```

Comment:
I believe that some additional steps are required here to make the application suitable for AppStore submission (e.g. removing `arm32` from the list of supported architectures).
Shall we add these steps to the tutorial?

Reply:
No. The build phase copy the same framework for the selected target on every run. That means we do not have merged universal framework here. There is no need to wipe anything before AppStore publication. Disclaimer - I did not yet publish anything

Reply:
TLDR; https://twitter.com/jonnyzzz/status/1045320269529845761
I base my sights on that post

Reply:
If deployment target is set to any version below 11.0, then Xcode can't build an application archive because the framework doesn't support arm32.
To override this behaviour, one can remove armv7* from the list of valid architectures in build settings.

Reply:
Thank you. I did not know that. Does that mean one has to have a universal binary with arm32 and arm64 to target to iOS < 11? 
It feels like too detailed to be included into text, but I'm happy to know that in detail, to be ready for possible questions

Reply:
> Does that mean one has to have a universal binary with arm32 and arm64 to target to iOS < 11?

Not exactly. As I have already mentioned above, it is possible to disable `arm32` support instead of making universal binaries, and this is likely to be enough to target to iOS < 11 on iPhone 5S or newer.